tests/sentry/api/endpoints/test_organization_details.py

```diff
@@ -799,6 +800,7 @@ def test_various_options(self, mock_get_repositories):
         assert "to {}".format(data["eventsMemberAdmin"]) in log.data["eventsMemberAdmin"]
         assert "to {}".format(data["alertsMemberWrite"]) in log.data["alertsMemberWrite"]
         assert "to {}".format(data["aiSuggestedSolution"]) in log.data["aiSuggestedSolution"]
+        assert "to {}".format(data["hideAiFeatures"]) in log.data["hideAiFeatures"]
```

Comment:
nit: we can additionally assert that the option `"sentry:hide_ai_feature"` is `True` (see L759-L773)